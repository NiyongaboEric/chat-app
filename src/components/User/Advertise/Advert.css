.advert-message {
  width: 40%;
  background-image: url('../../../Images/signup-chart.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center;
  height: 100vh;
}