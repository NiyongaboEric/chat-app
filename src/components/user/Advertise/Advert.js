import React from 'react';
import './Advert.css';

const Advertise = () => {
  return (
    <div className="advert-message">
    </div>
  );
};

export default Advertise;
