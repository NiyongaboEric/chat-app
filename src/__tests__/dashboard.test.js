import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Advert from '../components/user/Advertise/Advert';


Enzyme.configure({ adapter: new Adapter() });


describe('Advertisment', () => {
  const wrapper = shallow(<Advert />);
  it('Should advertise match snapshoot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Should advertise return length 0', () => {
    const text = wrapper.find('advert-message');
    expect(text.length).toBe(0);
  });

  it('Should look like', () => {
    const compareMarkup=<div className="advert-message" />;
    expect(wrapper.contains(compareMarkup)).toEqual(true);
  });

  it('Should advertise shallowed', () => {
    const text = wrapper.find('advert-message');
    expect(text.exists()).toEqual(false)
  });
});
