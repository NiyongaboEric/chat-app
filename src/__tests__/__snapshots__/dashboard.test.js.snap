// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Advertisment Should advertise match snapshoot 1`] = `ShallowWrapper {}`;
